test/fixture.rb

```diff
@@ -56,7 +62,7 @@ module Wrapper
     class BeforeAfterA
       def before_a(number)
         Fixtures.callstack.push([self.class, __method__, number])
-        number + 5
+        { args: [number + 5], kwargs: {} }
```

Comment:
This is the only breaking change I introduced in the PR. IMO we need to separate kwargs from positional args when returning values from `before_` methods in wrappers.